### ISSUE TITLE
Proposal: Docker image ready to run ansible

### ISSUE THREAD

**Issue by diegomanuel** [hashicorp/terraform-github-actions#165]
Hey folks, I'm not being able to run `ansible-playbook` from within a `provisioner "local-exec"` when running the `apply` command.

I had to manually add this "hack" into the `local-exec` to get access to `ansible` commands:
```
if ! [ -x "$(command -v ansible-playbook)" ]; then
  apk add --update --no-cache python3 ansible
fi
```

That's because `ansible` is not added to the [Dockerfile](https://github.com/hashicorp/terraform-github-actions/blob/master/Dockerfile#L3) :cry:

Is there any chance for you to add it in the near future?
`RUN ["/bin/sh", "-c", "apk add --update --no-cache bash ca-certificates curl git jq openssh python3 ansible"]`

Or may be a parameter like `tf_extra_image_packages` to give extra needed packages for the image?
```
- name: Terraform apply
        uses: hashicorp/terraform-github-actions@master
        with:
          tf_actions_version: 0.12.23
          tf_actions_subcommand: 'apply'
          tf_extra_image_packages: ["python3", "ansible"]
```

It would be nice to have it working out-of-the-box :wink: 
Thanks!

**Comment by sudomateo** [comment]
We're in the process of re-factoring this action into a JavaScript action. Once that is done, you'll be able to install Ansible a separate step irrespective of these Terraform actions.

**Comment by torgeir** [comment]
@sudomateo Any ETA on that initiative?

**Comment by canelkatmis** [comment]
Can you also please summorize why js is chosen

**Comment by mforutan** [comment]
@sudomateo, it would be very helpful if you can provide an ETA on this, we have difficulty using this GitHub Action, especially with `helm` provider as this doesn't respect any existing configuration from previous steps.
